app/presenters/tree_builder_automate_simulation_results.rb

```diff
@@ -71,9 +69,9 @@ def get_element_icon(el)
   def get_root_elements(el, idx)
     title = get_element_title(el)
     object = {:id          => "e_#{idx}",
-              :text        => _(title).html_safe,
+              :text        => title,
               :image       => get_element_icon(el),
-              :tip         => _(title).html_safe,
+              :tip         => title,
```

Comment:
If you trace back the variable, it is a string which isn't in the gettext catalog...do we need gettext calls for such strings? We agreed with @himdel that we don't.